scripts/maintenance/virsh-cleanup

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+CONNECT="${CONNECT:=qemu:///system}"
+POOL="${POOL:=default}"
+
```

Comment:
> nit: maybe also add NET="${NET:=tectonic}"?

Would this be for more targetted removal?  Currently the logic is just "destroy everything" (Tectonic or otherwise).